tests/by-util/test_cp.rs

```diff
@@ -1505,6 +1505,18 @@ fn test_copy_through_dangling_symlink() {
         .fails()
         .stderr_only("cp: not writing through dangling symlink 'target'");
 }
+#[test]
+fn test_copy_through_dangling_symlink_no_dereference() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    at.touch("file");
+    at.symlink_file("nonexistent", "target");
+    ucmd.arg("file")
+        .arg("-P")
+        .arg("target")
```

Comment:
According to https://github.com/uutils/coreutils/issues/3364#issue-1191201385=
it should be a bit different (and renamed to facilitate the match)

```suggestion
    at.symlink_file("no-such-file", "dangle");
    ucmd.arg("-P")
        .arg("dangle")
        .arg("d2")
```